framework/applications/noviusos_media/classes/renderer/folder.php

```diff
@@ -23,11 +23,23 @@ public function before_construct(&$attributes, &$rules)
 
     public function build()
     {
+        if (\Arr::get($this->renderer_options, 'multiple', 0)) {
+            $selected = array_map(function ($id) {
+                return array(
+                    'id' => $id,
+                    'model' => 'Nos\\Media\\Model_Folder',
+                );
+            }, (array)$this->value); // Casting to array in case the renderer is switched over from single to multiple
+        } else {
+            $selected = array(
+                // As a safety against changing from multiple to single, get the first value
+                'id' => is_array($this->value) ? reset($this->value) : $this->value,
+            );
+        }
         return $this->template(static::renderer(array(
             'input_name' => $this->name,
-            'selected' => array(
-                'id' => $this->value,
-            ),
+            'selected' => $selected,
+            'multiple' => \Arr::get($this->renderer_options, 'multiple', 0),
```

Comment:
Is there a reason to have `0` (int) as default value instead of `false` (bool) ?